advice.c

```diff
@@ -170,7 +170,10 @@ int error_resolve_conflict(const char *me)
 		 * other commands doing a merge do.
 		 */
 		advise(_("Fix them up in the work tree, and then use 'git add/rm <file>'\n"
-			 "as appropriate to mark resolution and make a commit."));
+			 "as appropriate to mark resolution and make a commit.\n"
+			 "In case of cherry-picking / reverting you can also use\n"
+			 "    git [cherry-pick | revert] --skip\n"
+			 "to skip the commit and continue."));
```

Comment:
I think this is currently only being used in `git cherry-pick`, or at least I couldn't quickly see where else this message would be displayed.  However it being used in the sequencer, I'm a bit reluctant to add `cherry-pick`/`revert` specific advice to it.  What if e.g. `rebase` uses the sequencer in the same way at some point?

Shouldn't it be parametrized based on `me` above?

Reply:
Oh yes! I'll use `me` to conditionally display this advice.

Reply:
Be careful with the commands for which you give the advice, e.g. `pull` doesn't have `--skip`.  I'll have to find some more time to actually read through where we give this advice, but maybe it's better to just err on the side of not adding this bit of advice here?

Reply:
Consider the following case:
```
$ git merge topic
$ git cherry-pick topic
```
In this one, there is no cherry-picking going but still, we will display the message `In case of ... use --skip ...` which is not correct. I guess we will have to leave this advice as is for now. Or is there any way of knowing if we are already cherry-picking/reverting?

Reply:
Also, these lines have got us covered.
https://github.com/r1walz/git/blob/536b750a74491d4d8e1f715408d622a4cb2e1083/sequencer.c#L2528-L2531
So, advising would not be so useful [here ](https://github.com/r1walz/git/blob/536b750a74491d4d8e1f715408d622a4cb2e1083/advice.c#L176-L178).

Reply:
Because this was force pushed, it looks like GitHub unfortunately doesn't allow me to see the context for this anymore, so I have to guess a bit what advice this was exactly.  But either way, I think just leaving this advice as it is is fine.

> Or is there any way of knowing if we are already cherry-picking/reverting?

`file_exists(git_path_cherry_pick_head(r))` and similar for `revert` should give us that information, no?

But yeah, overall I agree that this advice is probably overkill, and probably not necessary.

Reply:
> file_exists(git_path_cherry_pick_head(r)) and similar for revert should give us that information, no?

But we can't call this from `advice.c` (no ref. to `struct repository`)